### PR TITLE
Creating recipes article update

### PR DESCRIPTION
**Pull request by levshamin**

### CODE REVIEW

**Review by khanzadyan — APPROVED** [review]

File: content/_guides/creating-recipes.md [hunk 1]
```diff
@@ -5,166 +5,87 @@ section: Recipes
...
 ---
 
-This document provides information on how to [create Recipes](#creating-recipes), [manage Recipes](#managing-recipes), and how to [use them to create Flows](#activate-a-recipe---create-a-flow). Additionally, it gives a small glimpse into the new [Recipes UI](#coming-soon-recipes-ui) that is in development.
+This document provides information on how to [create Recipes from UI](#creating-recipes-from-ui), [create recipes using API](#creating-recipes-using-api) and how to [manage Recipes using API](#managing-recipes-using-api).
 
 [Recipes](/getting-started/recipes) allow users to share [Flow](/getting-started/integration-flow) templates with others without disclosing their non-shareable data ([Credentials](/getting-started/credential), Fields, Variables).
-
 ​
-## Creating Recipes
+## Creating Recipes from UI
```
Comment (HannaTrotsenko) [inline content/_guides/creating-recipes.md:16]: Rename to `Creating and Activating Recipes from UI`

File: content/_guides/creating-recipes.md [hunk 2]
```diff
@@ -5,166 +5,87 @@ section: Recipes
...
-  }'
+   {
+     "data": {
+       "visibility": "contract"
+     }
+   }'
 ```
 
-### Creating Recipes from Scratch
+> The **"visibility"** field can take the following values: **"workspace"**, **"contract"** or **"tenant"**.
```
Comment (HannaTrotsenko) [inline content/_guides/creating-recipes.md:45]: Workspace level means that Recipe available to be activated into current Workspace
Contract level means that Recipe available to be activated into any Workspace of current Contract
Tenant level means that Recipe available to be activated into any Contract of current Tenant.

Note: you can set visibility level of Recipe according to the Components visibility which used in Recipe.
Note: you can't downgrade Recipe visibility level

File: content/_guides/creating-recipes.md [hunk 3]
```diff
@@ -5,166 +5,87 @@ section: Recipes
...
-To create a Recipe from a chosen Flow, use the following request, provided you have `workspaces.recipe.edit` permission:
+After changing the visibility to **contract**, the recipe setup will look like this:
+
+![Contract visibility](/assets/img/integrator-guide/creating-recipes/contract-visibility.png)
+
+And after changing the visibility to **tenant**, the recipe setup will look like this:
+
+![Tenant visibility](/assets/img/integrator-guide/creating-recipes/tenant-visibility.png)
+
+  **3.** Here you can setup your credentials for every step. You can verify(**1**), edit(**2**) or delete(**3**) credentials and choose credentials you need(**4**):
```
Comment (luzhok) [inline content/_guides/creating-recipes.md:55]: also user can create a new credential!

File: content/_guides/creating-recipes.md [hunk 4]
```diff
@@ -5,166 +5,87 @@ section: Recipes
...
-| `relationships.workspace.data.id`   | yes          | Workspace ID               |
-| `relationships.workspace.data.type` | yes          | Allowed value: `workspace` |
+`POST /v2/recipes/`
 
+To see parameters and examples please visite [API-Documentation]({{site.data.tenant.apiBaseUri}}/docs/v2/#create-a-recipe-from-existing-flow).
 
-**EXAMPLE:**
+### Creating Recipes from Scratch
+
+To create a Recipe from a chosen Flow, use the following request, provided you have `workspaces.recipe.edit` permission:
```
Comment (luzhok) [inline content/_guides/creating-recipes.md:82]: sorry, from scratch or from a chosen flow ??? it looks, like the same, as the section above

File: content/_guides/creating-recipes.md (hunk 2 above)
Comment (khanzadyan) [reply content/_guides/creating-recipes.md:45]: done

File: content/_guides/creating-recipes.md (hunk 3 above)
Comment (khanzadyan) [reply content/_guides/creating-recipes.md:62]: done

File: content/_guides/creating-recipes.md (hunk 4 above)
Comment (khanzadyan) [reply content/_guides/creating-recipes.md:92]: corrected, done

**Review by HannaTrotsenko — APPROVED** [review]

**Pull request merged**